### PR TITLE
Fix #253 Add licenseMergesUrl also in report goals

### LINKED ISSUE #253: Add licenseMergesUrl also in report goals
Existing report goals:
- third-party-report
- aggregate-third-party-report

does not support licenseMergesUrl like their conterpart:
- add-third-party
- aggregate-add-third-party

### PR DESCRIPTION
**Pull request by elballa**
An integration test is available

### CODE REVIEW

File: src/main/java/org/codehaus/mojo/license/AbstractThirdPartyReportMojo.java [hunk 1]
```diff
@@ -345,6 +359,28 @@
...
+     * @throws Exception if any
+     */
+    protected void init()
+        throws Exception
+    {
+        if ( licenseMergesUrl != null )
+        {
+            if ( HttpRequester.isStringUrl( licenseMergesUrl ) )
+            {
+                InputStream input = new URL( licenseMergesUrl ).openStream();
```
Comment (ppalaga) [inline src/main/java/org/codehaus/mojo/license/AbstractThirdPartyReportMojo.java]: Let's use UrlRequester that is coming via #249 to do this.

Comment (elballa) [reply src/main/java/org/codehaus/mojo/license/AbstractThirdPartyReportMojo.java]: I'm waiting for merging of #249

Comment (ppalaga) [reply src/main/java/org/codehaus/mojo/license/AbstractThirdPartyReportMojo.java]: I just merged #249

**Comment by nfalco79** [comment]
@ppalaga do you prefer integrate this integration test with existing aggregate-third-party-report-license-merge with two executions or it's ok for a new integration test?

**Comment by ppalaga** [comment]
> do you prefer integrate this integration test with existing aggregate-third-party-report-license-merge with two executions or it's ok for a new integration test?

Same test project with multiple executions is faster to execute - so we all should clearly prefer it :)

**Comment by elballa** [comment]
Hi @ppalaga, sorry for my insistence, but do you have a plan for merging this PR?
Thanks

**Comment by ppalaga** [comment]
@elballa sorry, I have not noticed that you pushed. GH does not send e-mail notifications upon push (unless you add a comment). Thanks for the PR.

**Comment by nfalco79** [comment]
@ppalaga thank to you for the patient. Do you have any plan for a release?

**Comment by ppalaga** [comment]
Yeah, as soon as I finish things I work on, hopefully this week.

**Pull request merged**